modules/azure-private-dns/dns_zone.tf

```diff
@@ -12,4 +12,5 @@ resource "azurerm_private_dns_zone_virtual_network_link" "vnet_link" {
   resource_group_name   = var.resource_group_name
   private_dns_zone_name = azurerm_private_dns_zone.zone.name
   virtual_network_id    = each.value.vnet_id
+  registration_enabled = contains(keys(each.value), "registration_enabled") ? true : false
```

Comment:
can you enable this somewhere so we can see it in a plan?

Reply:
added to environments\sandbox\service-core-compute-idam-sprod-internal.yml